routers/web/repo/issue_suggestions.go

```diff
@@ -29,54 +26,11 @@ func IssueSuggestions(ctx *context.Context) {
 		isPull = optional.Some(false)
 	}
 
-	searchOpt := &issue_indexer.SearchOptions{
-		Paginator: &db.ListOptions{
-			Page:     0,
-			PageSize: 5,
-		},
-		Keyword:  keyword,
-		RepoIDs:  []int64{ctx.Repo.Repository.ID},
-		IsPull:   isPull,
-		IsClosed: nil,
-		SortBy:   issue_indexer.SortByUpdatedDesc,
-	}
-
-	ids, _, err := issue_indexer.SearchIssues(ctx, searchOpt)
```

Comment:
`issue_indexer.SearchIssues` does full-text search including issue content?

I am not sure whether it is good enough to only search title(name) by this PR's new approach.

Reply:
> `issue_indexer.SearchIssues` does full-text search including issue content?
> 
> I am not sure whether it is good enough to only search title(name) by this PR's new approach.

Looks like github did what I did but it will list all open issues/pulls

<img width="217" alt="image" src="https://github.com/user-attachments/assets/24b6fe51-5a30-46f8-ab61-45d96a3c5077" />

Reply:
OK. Github will search both title and content but the title-matchted issue will be listed first.

Reply:
> OK. Github will search both title and content but the title-matchted issue will be listed first.

Maybe GitHub uses search engine's ranking, not simply put something first.

Reply:
Maybe we can merge this first. Searching content is not usable from my side.